src/js/editor/editor.js

```diff
@@ -640,8 +642,27 @@ class Editor {
     return false;
   }
 
+  handleCopy(event) {
+    const { clipboardData } = event;
+    const range = this.cursor.offsets;
+    const mobiledoc = this.post.cloneRange(range);
+    const html = `<div data-mobiledoc='${JSON.stringify(mobiledoc)}'></div>`;
+    clipboardData.setData('text/html', html);
+    event.preventDefault();
```

Comment:
Aw man that would be cool. Opened a followup https://github.com/bustlelabs/content-kit-editor/issues/180